.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,24 +1,27 @@
-* **Do you want to request a *feature* or report a *bug*?**
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
 
+---
 
+**Describe the bug**
+A clear and concise description of what the bug is.
 
-* **What is the current behavior?**
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Run '....'
+4. See error
 
+**Expected behavior**
+A clear and concise description of what you expected to happen.
 
+**Desktop (please complete the following information):**
+ - OS: [e.g. Windows, Linux, Mac OSX]
+- Version: [e.g. Ubuntu 20.04]
 
```

Comment:
I would still add somewhere the TreeParzen version... I know we currently only have one sort of `beta` version but this will change at some point. It can be left it as it is until then but I guess it would help. Unless it is expected that users will always go for the latest.

Reply:
Added